units/systemd-sysupdate-reboot.service.in

```diff
@@ -0,0 +1,20 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=Reboot Automatically After System Update
+Documentation=man:systemd-sysupdate-reboot.service(8)
+ConditionVirtualization=!container
+
+[Service]
+Type=oneshot
+ExecStart={{ROOTBINDIR}}/systemd-sysupdate reboot
+
+[Install]
```

Comment:
Is this just here for convenience?  e.g. if you type `systemctl enable systemd-sysupdate-reboot.service` even though that won't do anything it _will_ enable the `timer`?

Reply:
yes